ZUGFeRD/InvoiceDescriptor.cs

```diff
@@ -702,5 +749,25 @@ public void AddDebitorFinancialAccount(string iban, string bic, string id = null
                 BankName = bankName
             });
         } // !AddDebitorFinancialAccount()
-   }
+
+        private string GetNextLineId()
+        {
+            string _lastLineID = "0";
+            int _lastnumericLineID = 0;
+
+            if (this.TradeLineItems.Count > 0)
+            {
+                _lastLineID = this.TradeLineItems.Last().AssociatedDocument.LineID;
```

Comment:
would it make sense to find the highest current number by iterating/ or by linq?

Reply:
this would be also a possible solution, you can change this behavior if you want. I tried the simple way here :-)